### DISCUSSION TITLE
Add HA entities

### DISCUSSION THREAD

**Discussion by thepuncake** [blakadder/nspanel#59] [General]
Hi there and first things first congrats on cracking this nut to blackadder.

Will there eventually be a way to add HA entities in the views of the nspanel? As I understand it can now show/switch only tasmota devices or did I get it wrong?

**Comment by joBr99** [comment]
you need to build your own automations to translate between nspanel and and homeassistant

or just use one of the projects made for home assistant

https://github.com/Blackymas/NSPanel_HA_Blueprint
https://github.com/joBr99/nspanel-lovelace-ui

**Reply by thepuncake** [reply]
Thank you for your answer. If I use the esphome component while on tasmota could I run the blackymas blueprint for example? Otherwise I have to use appdaemon correct?

**Reply by joBr99** [reply]
blackymas is esphome only with an HA Automation to translate between esphome and homeassistant

nspanel-lovelace-ui is tamota or esphome with an appdaemon app to translate between homeassistant and tasmota
